docs/permission/writing-a-policy.md

```diff
@@ -0,0 +1,5 @@
+---
+id: writing-a-policy
+title: Writing a permission policy
+description: How to write your own permission policy as a Backstage integrator
```

Comment:
nit: do we mean the same thing by "adopter" in the  getting started description and "Backstage integrator" here? If so, perhaps we should pick one and use it for both.

Reply:
Is it necessary to specify the persona here? Could just leave it at "How to write your own permission policy"

Reply:
Good catch - sticking with integrator for now, but we can find all + replace later if we decide differently.